script.js

```diff
@@ -0,0 +1,82 @@
+window.addEventListener('load', init);
+document.addEventListener('keydown', print);
+document.addEventListener('keyup', () => {
+    document.querySelectorAll('.keyboard__key').forEach(element => {
+        if (element.innerText != 'CapsLock') element.classList.remove('keyboard__key--active');
```

Comment:
for better code readibility it's preferably to wrap one line if statements with curly braces as well

Reply:
Also, taking into account that 'keyboard_key--active' class name is used a few times, it's better to use variable with it. It will reduce probability of error because of misspelling